ironfish-rust/Cargo.toml

```diff
@@ -43,7 +43,7 @@ chacha20poly1305 = "0.10.1"
 crypto_box = { version = "0.9", features = ["std"] }
 ff = "0.12.0"
 group = "0.12.0"
-ironfish-frost = { git = "https://github.com/iron-fish/ironfish-frost.git", branch = "main" }
+ironfish-frost = { git = "https://github.com/iron-fish/ironfish-frost.git", branch = "main", features = ["std", "signing"] }
```

Comment:
Do we want to use the default features here, so we don't need to define custom features? The difference I guess is that ironfish-frost has dkg, but we aren't enabling it here https://github.com/iron-fish/ironfish-frost/blob/main/Cargo.toml#L27

Reply:
Yeah I think we can use the default features here. We also support dkg in the CLI, and I can't remember what's behind the dkg feature in ironfish-frost